### ISSUE TITLE
layout should be initialized smartly for 2D images

### ISSUE THREAD

**Issue by ebrahimebrahim** [Kitware/VolView#397]
When loading a 2D image into volview, we get a single slice and in this case the quad view is not useful. Many users might not even see that they can change the layout, and their first impression of volview could then easily be "oh, this is not for use with xrays, okay bye". It might make sense to automatically switch to an appropriate layout when the loaded volume appears to be a 2D image / single slice.

Just an idea -- feel free to close if it's not in line with the vision of how layouts work or how volview is meant to be used :slightly_smiling_face:

**Comment by floryst** [comment]
Agreed! This is related to #375 and #330. The goal is to have a system for auto-picking layouts for datasets based on various factors (e.g. reconstructability).
